doc/constraint_causal_discovery.rst

```diff
@@ -0,0 +1,136 @@
+.. _constraint_causal_discovery:
+
+==================================
+Constraint-based causal discovery
+==================================
+.. automodule:: dodiscover.constraint
+    :no-members:
+    :no-inherited-members:
+.. currentmodule:: dodiscover
+
+The following are a set of methods intended for (non-parametric) structure learning
```

Comment:
Referencing to my previous comment in the CI part, here you could instead mention that CI correspond to d-separation in a graph etc.

Reply:
Done